composer.json

```diff
@@ -74,6 +75,12 @@
         "sort-packages": true
     },
     "extra": {
+        "composer-exit-on-patch-failure": true,
+        "patches": {
+            "twig/twig": {
+                "Allow Traversable objects to be used in the with tag": "https://github.com/twigphp/Twig/commit/650bc7c262d0c84ece3c472c9302bf416475f72b.patch"
```

Comment:
Not ideal as this isn't the merged change, will update when it appears in the Twig `2.x` branch. (Then remove entirely when it's released.)

Reply:
It's now there, but can't seem to get GitHub to produce a patch file that applies. Will just wait for the next release.